tool/build.dart

```diff
@@ -293,22 +338,21 @@ _main(List<String> argv) async {
         }
       }
 
-      var pkgs = readPackages(_Packages);
+      var pkgs = await readPackages(_Packages);
```

Comment:
These method calls must be asynchronous as they call fail() which is asynchronous.